misk/src/main/kotlin/misk/web/BoundAction.kt

```diff
@@ -148,3 +153,28 @@ internal class RequestMatch(
 
 private fun MediaType.closestMediaRangeMatch(ranges: List<MediaRange>) =
     ranges.mapNotNull { it.matcher(this) }.sorted().firstOrNull()
+
+/** Acts as the bridge between network interceptors and application interceptors.
```

Comment:
supernit: we like the `/**` to be on a line of its own, and only once space after the `*` on each line beneath

Reply:
(or perhaps, _I_ like!)